environments/icds-new/postgresql.yml

```diff
@@ -1,6 +1,7 @@
 DEFAULT_POSTGRESQL_HOST: pgmain1
 REPORTING_DATABASES:
   ucr: ucr
+  icds-ucr-non-dashboard: icds-ucr
```

Comment:
I know this is perhaps not needed, but updated since it exists and since UCRs have this as an env.